### PR TITLE
Small changes to account page

### PR DESCRIPTION
**Pull request by Lukewh**
# Done

- Removed account column
- Added snap icon next to name
- Added 'Private' next to name of private snaps

# QA

- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004/account or [demo url]/account
- Make sure the page still works

# Screenshot

screenshot from 2018-02-15 14-32-19

### CODE REVIEW

File: modules/publisher/views.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def get_account():
     if '16' in account['snaps']:
         user_snaps = account['snaps']['16']
 
+    print(account)
```
Comment (tbille) [inline modules/publisher/views.py]: hum hum

File: templates/account.html [hunk 2]
```diff
@@ -44,15 +43,22 @@ <h3>My snaps</h3>
                   {% else %}
                     {% for snap in user_snaps|sort %}
                         <tr>
-                            <td>
+                            <td class="u-vertically-center">
+                                {% if user_snaps[snap].icon_url %}
+                                    <img src="{{ user_snaps[snap].icon_url }}" width="32" height="32" />&nbsp;
```
Comment (tbille) [inline templates/account.html]: `width="32" height="32"` could it be in a css file instead

File: modules/publisher/views.py (hunk 1 above)
Comment (Lukewh) [reply modules/publisher/views.py]: whoops, good spot, will remove

File: templates/account.html (hunk 2 above)
Comment (Lukewh) [reply templates/account.html]: There isn't any css for this page and it's getting redesigned so I don't think it's worth creating a new css file just for this - vanilla all the way.

**Comment by Greggless** [comment]
sexy ☁️ 🚀 ☁️

**Comment by Lukewh** [comment]
Thanks everyone! Yeah @Greggless I'm going to dig in to this as it'd be good to show whether the snap is a snap you're a collaborator on, or you own!

**Comment by bartaz** [comment]
Not sure if we can fix this without some custom styles, but icon feels too close to snap name, and snap name bit too close to 'Private' text.

**Pull request merged**